src/libLookup/Lookup.cpp

```diff
@@ -3662,7 +3659,48 @@ void Lookup::SenderTxnBatchThread() {
   DetachedFunction(1, main_func);
 }
 
-void Lookup::SendTxnPacketToNodes(uint32_t numShards) {
+void Lookup::RectifyTxnShardMap(const uint32_t oldShardSize,
```

Comment:
For consistency with other function, might be better to say `oldNumShards` and `newNumShards`